.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,15 +1,25 @@
-<!--
-Thanks for reporting a problem with Airsonic! Please complete all of the
-following sections to make it easier to assist you.
--->
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'in: UNKNOWN-frontend, status: waiting-for-feedback, type: bug'
+assignees: ''
+
+---
 
 ## Problem description
 
-*Describe your problem here. Describe what you want to happen, and what happens
+*Describe your problem here. Describe what you want to happen, and what happens 
 if you try to do it. If you have a stack trace or any logs, please format them using
 github's triple-backquote notation, or if they are large you may also consider
 attaching them as a file or gist.*
 
+### Please select a label
+
+*Of the labels displayed on the right, change the label that starts with 'in:' to an appropriate one.
```

Comment:
Can regular users assign labels when they are not part of the maintainers?

Reply:
This needs some confirmation. I have not logged out and tried.

Reply:
I tried. There is no label. :relaxed:
![image](https://user-images.githubusercontent.com/27724847/82221024-63482400-995a-11ea-9d36-8830dd4dab00.png)